pkg/kubectl/apps/kind_visitor.go

```diff
@@ -79,11 +79,26 @@ type NoOpKindVisitor struct{}
 
 var _ KindVisitor = &NoOpKindVisitor{}
```

Comment:
/hold
`NoOpKindVisitor{}` is technically unused.
so it seems the is no need to define it and it's methods.

`TestKindVisitor` already implements the interface for doing unit testing.

my vote would be to remove it, unless there are plans for it?
/assign @soltysh @pwittrock

Reply:
Thanks @neolit123 
If it is useless, I'd be happy to clean it up.

Reply:
/hold cancel